dashboard/consts.py

```diff
@@ -26,6 +26,7 @@
 GCS_RETRY_CONNECT_INTERVAL_SECONDS = env_integer(
     "GCS_RETRY_CONNECT_INTERVAL_SECONDS", 2
 )
+GCS_RPC_TIMEOUT = 3
```

Comment:
If this value is in seconds, can we call this variable `GCS_RPC_TIMEOUT_SECONDS` to match the other constants' formats?

Reply:
Sure. I'll do. I fail to add seconds because in py it seems like it's seconds by default. but i think following the formats are more important.